src/odemis/util/registration.py

```diff
@@ -531,11 +535,15 @@ def estimate_grid_orientation_from_img(
     -------
     tform : instance of `transform_type`
         The orientation of the pattern.
+    error_metric : float
+        The RMS value of the fiducial error registration of the square grid
+        compared to the spots measured on the image.
 
     """
     if num_spots is None:
         num_spots = shape[0] * shape[1]
     elif num_spots == 0:
         num_spots = None
     ji = find_spot_positions(image, sigma, threshold_abs, threshold_rel, num_spots)
-    return estimate_grid_orientation(ji, shape, transform_type)
+    tform, error_metric = estimate_grid_orientation(ji, shape, transform_type)
+    return tform, error_metric
```

Comment:
No need to modify? the old `return estimate_grid_orientation(ji, shape, transform_type)` would still work here.

Reply:
Good catch, I'll change it back